spec/api-browser-window-spec.js

```diff
@@ -256,6 +256,14 @@ describe('BrowserWindow module', function () {
         w.loadURL(server.url)
       })
 
+      it('should support support base url for data urls', (done) => {
+        ipcMain.once('answer', function (event, test) {
+          assert.equal(test, 'test')
+          done()
+        })
+        w.loadURL('data:text/html,<script src="loaded-from-dataurl.js"></script>', {baseURLForDataURL: 'file://' + path.join(fixtures, 'api') + path.sep})
```

Comment:
Yes, the path separator is required, otherwise it will not be able to load the scrip. Also, one quick thing, I noticed that I have put this test inside "POST navigations". Is that alright, or should I move it into it's own test?